modules/llm-cache/ds/kv_state_cache_manager.cc

```diff
@@ -400,12 +404,25 @@ Status KVStateCacheManager::ClearGlobalCache(Client& client,
                                        config.llmRefcntObjectName);
 }
 
-Status ClearGlobalCache(Client& client, FileCacheConfig& config) {
-  // TBD
+void KVStateCacheManager::Close() { storage->CloseCache(); }
+
+Status KVStateCacheManager::StopGlobalGCThread() {
+  if (this->type != CacheType::FILE) {
+    return Status::Invalid("Only support for file storage");
+  }
+
+  std::dynamic_pointer_cast<FileStorage>(storage)->StopGlobalGCThread();
   return Status::OK();
 }
 
-void KVStateCacheManager::Close() { storage->CloseCache(); }
+Status KVStateCacheManager::StartGlobalGCThread() {
+  if (this->type != CacheType::FILE) {
+    return Status::Invalid("Only support for file storage");
+  }
+
+  std::dynamic_pointer_cast<FileStorage>(storage)->StartGlobalGCThread();
+  return Status::OK();
+}
```

Comment:
It is a bit tedious (and misleading) to have a new config for storage type, imagining you have a manager whose type is blob but the storage is file storage (the current code has no way to prevent such a combination).

You would better implement `StartGlobalGCThread` to blob storage as well (as an empty function) and here you could just call `storage->StartGlobalGCThread()`.

Reply:
OK